lib/edit.js

```diff
@@ -22,6 +22,20 @@ function edit (args, cb) {
     ))
   }
   p = p.split('/')
+    // combine scoped parts
```

Comment:
Ordinarily I'd ask for tests, but we don't have any existing tests for this route, this is a interactive route and this change is reasonably eyeballable, so I'm inclined to take this without them.

Reply:
Yeah, I tried making tests but I'm not too familiar with all the mocking environments yet, and without something to work with I wasn't sure how to.